index.bs

```diff
@@ -325,8 +321,16 @@ interface PictureInPictureWindow {
 };
 </pre>
 
-The {{PictureInPictureWindow}} is a simple proxy to get information such as
-dimensions about the current Picture-in-Picture window.
+A {{PictureInPictureWindow}} instance represents a <a>Picture-in-Picture
+window</a>. It has a |state| that is either |opened| or |closed|.
+
```

Comment:
You should probably mention that a PIPWindow is associated to an element?

Reply:
Done.